packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -619,6 +619,20 @@ export default class BackupNg {
         const jobId = job.id
 
         const remoteIds = unboxIdsFromPattern(job.remotes)
+
+        await Promise.all(
+          remoteIds.map(async id => {
+            const remote = await app.getRemote(id)
+            if (remote.proxy !== job.proxy) {
+              throw new Error(
+                job.proxy !== undefined
+                  ? `The remote ${remote.name} must be linked to the proxy ${job.proxy}`
+                  : `The remote ${remote.name} must not be linked to a proxy`
+              )
+            }
+          })
+        )
+
```

Comment:
So what about [this](https://github.com/vatesfr/xen-orchestra/pull/4907#discussion_r412286738)?